templates/web/base/reports/_ward-list.html

```diff
@@ -1,16 +1,27 @@
 [% IF children.size %]
 <section id="council_wards" class="hidden-js">
 <h2>[% ward_text %]</h2>
-    <p>[% loc('Follow a ward link to view only reports within that ward.') %]</p>
+    <p class="js-ward-single">[% loc('Follow a ward link to view only reports within that ward.') %]</p>
+    <p class="js-ward-multi hidden">[% loc('Select multiple wards to view only reports within those wards.') %]</p>
+    <form method="get" action="">
     <section class="full-width">
     <ul class="item-list item-list--wards">
       [% FOR child IN children.values.sort('name') %]
         <li class="item-list__item item-list--wards__item">
-            <a href="[% child.url %]">[% child.name %]</a>
+          <label class="js-ward-multi hidden">
+            <input type="checkbox" name="ward" value="[% child.name | html %]">
+            [% child.name %]
+          </label>
+          <a class="js-ward-single" href="[% child.url %]">[% child.name %]</a>
         </li>
       [% END %]
     </ul>
     </section>
+    <p>
+      <input class="btn js-ward-multi hidden" type="submit" value="[% loc('View wards') %]" />
+    </p>
+    <p class="js-ward-single"><a href="#" class="js-ward-select-multiple">[% loc('Select multiple wards?') %]</a></p>
```

Comment:
Can we have this at the top? Bit hard to find at the bottom.

Also the button shouldn't be viewable unless JS is enabled – or actually *ideally*, the button would submit to the server and reshow the page with the checkboxes shown, please :)

Reply:
How about f26feef26d8a1961b124869d47aa4c749f25c5af? :)

Reply:
That'll do nicely!